src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -437,22 +437,22 @@ public static void Constructor_String_LoadVersionTest()
             string assemblyNamePrefix = "System.Reflection.Tests.Assembly_";
 
             // Requested version 1.0 does not load 0.0.0.0, but loads 1.2.0.0, 3.0.0.0
-            Assert.Throws<FileLoadException>(() => Assembly.Load(new AssemblyName(assemblyNamePrefix + "0_0_0_0, Version=1.0")));
+            Assert.Throws<FileNotFoundException>(() => Assembly.Load(new AssemblyName(assemblyNamePrefix + "0_0_0_0, Version=1.0")));
```

Comment:
@AtsushiKan @GrabYourPitchforks I had to change the expected exception here to FNFE. There's a chance that these tests never run (custom xunit fork bug) before. Please let me know if that's now indeed the expected behavior.

Reply:
LGTM